src/components/modal/v2/parts/views/LongTerm/Content.jsx

```diff
@@ -144,26 +145,44 @@ export const LongTerm = ({
         return null;
     };
 
+    // New checkout modal designs utilize a sticky button
+    const conditionalStickyButton =
+        useNewCheckoutDesign === 'true' ? (
+            <div className="button__fixed-wrapper">{renderCheckoutCtaButton()}</div>
+        ) : (
+            renderCheckoutCtaButton()
+        );
+
     return (
         <Fragment>
             <style>{styles._getCss()}</style>
-            <div className="content__row dynamic">
+            <div className={`content__row dynamic ${useNewCheckoutDesign === 'true' ? 'checkout' : ''}`}>
                 <div className="content__col">
                     <Calculator
                         setExpandedState={setExpandedState}
                         calculator={calculator}
                         aprDisclaimer={offerAPRDisclaimers}
                         useV4Design={useV4Design}
+                        useNewCheckoutDesign={useNewCheckoutDesign}
                     />
                     <div className={`content__col ${expandedState ? '' : 'collapsed'}`}>
                         <div className="branded-image">
                             {/* TODO: include Icon component when desktop images are final */}
                         </div>
                     </div>
                 </div>
-                <Instructions instructions={instructions} useV4Design={useV4Design} expandedState={expandedState} />
+                <Instructions
+                    instructions={instructions}
+                    useV4Design={useV4Design}
+                    useNewCheckoutDesign={useNewCheckoutDesign}
+                    expandedState={expandedState}
+                />
             </div>
-            <div className={`content__row disclosure ${expandedState ? '' : 'collapsed'}`}>
+            <div
+                className={`content__row disclosure ${expandedState ? '' : 'collapsed'}  ${
```

Comment:
```suggestion
                className={`content__row disclosure ${expandedState ? '' : 'collapsed'} ${
```
Small spacing nitpick.